packages/luna/src/components/StudentOverview/StudentOverview.tsx

```diff
@@ -72,7 +75,7 @@ export const StudentOverview: React.FC<Props> = ({
     );
 
   return (
-    <div className="tw-flex tw-gap-6 tw-flex-wrap tw-w-full">
+    <div className="tw-grid tw-grid-rows-2 tw-grid-cols-2 md:tw-flex tw-gap-4 md:tw-gap-6 tw-flex-wrap tw-w-full">
       <Card
         icon={<Video16X16 className="[&>*]:tw-stroke-natural-50" />}
```

Comment:
I would like to take the chance to fix these icons. The reason we name it like this `Video16x16` is we didn't know how to set the width dynamically using a single icon. This is can be done by doing this. 

```diff
- <svg width="16" height="16" viewBox="0 0 16 16" fill="none"
+ <svg width="100%" height="100%" viewBox="0 0 16 16" fill="none"
```

After your change, we should only have one icon called `video`.


```suggestion
        icon={<Video16X16 className="[&>*]:tw-stroke-natural-50 tw-w-4 tw-h-4" />}
```

> [!NOTE]
> Remember to update the other icons in this file as well.